lib/plugins/renderer/nunjucks.js

```diff
@@ -0,0 +1,61 @@
+
+'use strict';
+
+const nunjucks = require('nunjucks');
+const fs = require('hexo-fs');
+
+function toArray(value) {
+  if (Array.isArray(value)) {
+    // Return if given value is an Array
+    return value;
+  } else if (typeof value.toArray === 'function') {
+    return value.toArray();
+  } else if (value instanceof Map) {
+    const arr = [];
+    value.forEach(v => arr.push(v));
+    return arr;
+  } else if (value instanceof Set || typeof value === 'string') {
+    return [...value];
+  } else if (typeof value === 'object' && value instanceof Object && Boolean(value)) {
+    return Object.values(value);
+  }
+
+  return [];
+}
+
+function safeJsonStringify(json, spacer = undefined) {
+  if (typeof json !== 'undefined' && json !== null) {
+    return JSON.stringify(json, null, spacer);
+  }
+
+  return '""';
+}
+
+const env = nunjucks.configure({
```

Comment:
I guess it's related to the first param (`path`) of `nunjucks.configure`: https://mozilla.github.io/nunjucks/api.html#configure

> path defaults to the current working directory

Leaving it empty might cause relative path resolving issues.